Upload scripting

MitchellGerdisch opened the pull request:
### Description

Provides a script to upload and publish the policy CATs to a RightScale account.
It relies on a "master_policy.list" file that lists the package files and CAT files.
So as new CATs are added and/or new package files added/imported, they need to be added to the applicable part of the master_policy.list file.

### Issues Resolved

Makes it easy to push the policy CATs to a customer's account.

### Contribution Check List

- [X ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

srpomeroy commented:
@MitchellGerdisch We typically recommend NOT publishing CATs so that only users with the `ss_designer` role can see/run them.

Per the main read me:
> Launch the CAT in Test mode from the Designer screen of RightScale Self-Service using your desired schedule. Running in Test mode from the Designer screen means that only other users with the Designer role can see or run the policy CATs and it will not show in the Self-Service Catalog for other users. Do not choose Alert and Delete until you are confident you know what will be deleted.

May be worth adding an option for publish. Especially because we usually run them on a schedule, so if we do publish, we add a schedule.

rshade commented:
also if you do not mind can you stick them in a tools folder

MitchellGerdisch commented:
Moved to a tools folder
Added optional -p parameter to specify if you want to publish.
Added optional -s parameter to specify a schedule with which to publish
